frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -46,9 +42,18 @@ import { GuestAgentInfoWrapper } from '../../k8s/wrapper/vm/guest-agent-info/gue
 import { VMStatusBundle } from '../../statuses/vm/types';
 import { NOT_AVAILABLE_MESSAGE } from '../../strings/vm/messages';
 import { isGuestAgentInstalled } from '../dashboards-page/vm-dashboard/vm-alerts';
+import { getGuestAgentFieldNotAvailMsg } from '../../utils/guest-agent-strings';
+import { Button } from '@patternfly/react-core';
+import {
+  isFlavorChanged,
+  isCDROMChanged,
+  isBootOrderChanged,
+} from '../../selectors/vm-like/nextRunChanges';
 
 import './vm-resource.scss';
-import { getGuestAgentFieldNotAvailMsg } from '../../utils/guest-agent-strings';
+import { VMWrapper } from '../../k8s/wrapper/vm/vm-wrapper';
+import { VMIWrapper } from '../../k8s/wrapper/vm/vmi-wrapper';
+import { K8sResourceKind } from '../../../../../public/module/k8s/types';
```

Comment:
`import './vm-resource.scss' should be the last import

Reply:
removed it and used a class from _common as you suggested